src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: start;
+}
+
+.stars__star {
+  display: flex;
+  justify-content: center;
+  background-image: url(images/star.svg);
```

Comment:
Consider moving the common property `background-image: url(images/star.svg);` to the `.stars__star` class. This way, you will have less code and it will be easier to manage.